### DISCUSSION TITLE
Halving BPE vocab size does not reduce mem usage ???

### DISCUSSION THREAD

**Discussion by iggygeek** [k2-fsa/icefall#1597] [Q&A] [answered]
Hello ! I compared two runs with the Librispeech zipformer2 recipe (with my dataset)  with BPE vocab size = 500 and 250. However the GPU memory usage is ~9.5GB for both runs. Is that expected ?
Other parametres like max-duration are the same...

**Comment by JinZr** [comment, marked as answer]
yes that’s expected, changing the BPE vocab size only alters the number of parameters of the last layer, the difference is too trivial to be reflected on GPU memory usage. bestjin On 17 Apr 2024, at 19:32, iggygeek ***@***.***> wrote:﻿
Hello ! I compared two runs with the Librispeech zipformer2 recipe (with my dataset)  with BPE vocab size = 500 and 250. However the GPU memory usage is ~9.5GB for both runs. Is that expected ?
Other parametres like max-duration are the same...

**Comment by iggygeek** [comment]
Yes perfectly clear
